src/test/resources/grouper.test.properties

```diff
@@ -83,4 +83,11 @@ ws.assign.grouper.privileges.results.success={ "resultMetadata": { "params": nul
 ws.group={ "idIndex": "835554", "typeOfGroup": "group", "extension": "extension", "displayExtension": "extension", "description": "description", "displayName": "group-path", "name": "group-path", "uuid": "grouper-uuid", "detail": { "hasComposite": "T", "leftGroup": null, "rightGroup": null, "typeNames": [], "attributeNames": null, "attributeValues": null, "compositeType": "complement", "params": null, "createSubjectId": "_uhims_grouper", "createTime": "2019/02/04 15:26:03.369", "isCompositeFactor": "F", "modifySource": null, "modifySubjectId": "_groupings_api_2", "modifyTime": "2023/04/03 11:30:49.113" } }
 
-ws.empty.results={"results": []}
\ No newline at end of file
+ws.empty.results={"results": []}
+
+ws.assign.attributes.results.changed.true={"wsAttributeAssignResults":[{"changed":"T","deleted":"F"}],"wsAttributeDefNames":[{"name":"name"}],"resultMetadata":{"resultCode":"SUCCESS","success":"T"}}
+ws.assign.attributes.results.changed.false={"wsAttributeAssignResults":[{"changed":"F","deleted":"F"}],"wsAttributeDefNames":[{"name":"name"}],"resultMetadata":{"resultCode":"SUCCESS","success":"T"}}
+
+
```

Comment:
I would remove these two newlines here because these four new test properties should be all grouped together.